### PR TITLE
Add more information on ranges in Haskell

### PR DESCRIPTION
**Pull request by devinmcginty**

### CODE REVIEW

File: haskell.html.markdown [hunk 1]
```diff
@@ -67,10 +67,18 @@ not False -- True
 ----------------------------------------------------
 
 -- Every element in a list must have the same type.
--- Two lists that are the same
+-- These two lists are the same
```
Comment (vendethiel) [inline haskell.html.markdown]: do you mind adding a colon? :)

File: haskell.html.markdown [hunk 2]
```diff
@@ -67,10 +67,18 @@ not False -- True
...
 [1, 2, 3, 4, 5]
 [1..5]
 
+-- Ranges are versatile.
+['A'..'F'] -- "ABCDEF"
```
Comment (vendethiel) [inline haskell.html.markdown]: is the fact that String == [] Char` explained before?

Comment (devinmcginty) [inline haskell.html.markdown]: It was mentioned, but somewhat easy to miss. I added a second example of a string as a list.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**